src/main/java/coydir/logic/commands/BatchAddCommand.java

```diff
@@ -0,0 +1,124 @@
+package coydir.logic.commands;
+
+import static coydir.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import coydir.logic.commands.exceptions.CommandException;
+import coydir.logic.parser.AddCommandParser;
+import coydir.logic.parser.exceptions.ParseException;
+import coydir.model.Database;
+import coydir.model.Model;
+import coydir.model.person.Person;
+
+/**
+ * Adds multiple person to the database
+ */
+public class BatchAddCommand extends Command {
+    public static final String COMMAND_WORD = "batchadd";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds multiple people into the database from a csv file \n"
+            + "Parameters: filename (must be in the data folder of the repository and CSV format)\n"
+            + "Example: " + COMMAND_WORD + " coydir.csv";
+    public static final String MESSAGE_SUCCESS = "Batch Add Success. %d employees were added";
+
+    private static final String[] PREFIX_LIST = { "n/", "p/", "e/", "j/", "a/", "t/"};
+    private final String filename;
+    private Path filePath;
+
+    /**
+     * Creates an BatchAddCommand to add the multiple {@code Person}
+     */
+    public BatchAddCommand(String filename) {
+        requireAllNonNull(filename);
+        this.filename = filename;
+        this.filePath = Paths.get("data", this.filename);
+    }
+
+    public void setFilePath(Path filePath) {
+        this.filePath = filePath;
+    }
+
+    public List<AddCommand> getInfo() throws CommandException {
+        Path file = this.filePath;
+        String line = "";
+        String splitBy = ",";
+        List<AddCommand> addCommandList = new ArrayList<>();
+        try {
+            // parsing a CSV file into BufferedReader class constructor
+            BufferedReader br = new BufferedReader(new FileReader(file.toString()));
+            br.readLine();
+            while ((line = br.readLine()) != null) {
+                String[] data = line.split(splitBy); // use comma as separator
+                String arg = " ";
+                for (int i = 0; i < data.length; i++) {
+                    if (i == (data.length - 1)) {
+                        String[] tags = data[i].split("/");
+                        for (String tag : tags) {
+                            arg += PREFIX_LIST[i] + tag + " ";
+                        }
+                    } else {
+                        arg += PREFIX_LIST[i] + data[i] + " ";
+                    }
+                }
+                addCommandList.add(new AddCommandParser().parse(arg));
+            }
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
Applies for line 77 below also.

Can we not print the error stack? It kinda ruins the logging in the terminal.

![image](https://user-images.githubusercontent.com/78629832/196036959-e89d244c-1e6d-4a93-b7ff-7a6eef5e9ed0.png)